docker/set_up_sentry_cli.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Installs sentry-cli.
+
+set -v -e -x
+
+# Comes from https://sentry.io/get-cli/
+SENTRY_URL=https://github.com/getsentry/sentry-cli/releases/download/1.32.0/sentry-cli-Linux-x86_64
```

Comment:
If they ever do an update, we'll continue to get this version. We don't have a good way of knowing whether a new version has come out. I'm not sure that's a big deal, but it might be at some point in time.

Maybe a "better" way is to get the script at https://sentry.io/get-cli/ and then parse the download line and then curl it?

Reply:
Apparently we can get it from node as well: https://github.com/getsentry/sentry-cli#node. I'm not sure if that form is compatible with our auto-updates, hashing, or vulnerability-checking, though. If not, it's not terribly different from using this URL.

Reply:
I thought about adding it to `webapp-django/package.json`. That'd use our existing infrastructure for notifying us there are new versions. However, I'm concerned about how we install a bunch of JS libs for different purposes that run in different contexts all at once. I didn't want to add the sentry cli to the webapp JS lib mix since it's only used in the crontabber container.

Looking at this more, I wonder if it's possible for us to have a `crontabber/package.json` file. Then it's separate from the webapp bits and we get our version update notification. Will that work with whatever is telling us our versions are out of date?